docs/docs/api/CacheStore.md

```diff
@@ -21,27 +22,32 @@ The store must implement the following functions:
 
 ### Getter: `isFull`
 
-This tells the cache interceptor if the store is full or not. If this is true,
+Optional. This tells the cache interceptor if the store is full or not. If this is true,
 the cache interceptor will not attempt to cache the response.
 
-### Function: `createReadStream`
+### Function: `getValueByKey`
```

Comment:
Any reason it can't be just `get`?

Reply:
Personal preference, but either or work for me

Reply:
Seem unnecessarily verbose to me...